architecture/additional_concepts/flannel.adoc

```diff
@@ -29,4 +31,9 @@ responsibile for:
 
 Each *flanneld* agent provides this infomation to a centralized *etcd* store so
 other agents on hosts can create an overlay network and route packets to
-any container within the *flannel* network.
\ No newline at end of file
+any container within the *flannel* network.
+
+The following diagram illustrates the architecture and data flow from one
+container to another using a *flannel* network:
+
+image::flannel.png[Flannel Communication]
```

Comment:
@rajatchopra -- I've got the image updated along with the text to accommodate your feedback. I've put the example routes underneath the image, rather than within the image itself.

How does it all look now?